golangci.yml

```diff
@@ -81,7 +82,6 @@ linters:
     - mnd
     - testpackage
     - wrapcheck
```

Comment:
Commenting a random line

You are using enable-all, but this setting is planned to be removed.

As you are working on the file, maybe you could review it now.

Reply:
> You are using enable-all, but this setting is planned to be removed.

?

Reply:
My bad, they changed their mind

https://github.com/SeigeC/golangci-lint/commit/4d97f054fb5726e4c1d392b66b7926c344bcaeaf